docs/Community/conduct/README.md

```diff
@@ -0,0 +1,18 @@
+Community Conduct
+=================
+
+The Open Enclave SDK Project Community strives to be an inclusive, welcoming,
+and friendly place for all contributors.
+
+All community members agree to abide by the
+[Code of Conduct](../../CODE_OF_CONDUCT.md), and we look to the community leaders to
+uphold the [Project's Core Values](../Governance.md#Core-Principles).
+
+We acknowledge that sometimes minor disagreements may occur, and have written
+down our general views on how conflict between community members
+[should be handled](ConflictPolicy.md).
+
+Incidents may be reported to the
```

Comment:
```suggestion
Incidents should be reported to the
```

I want somehow to encourage folks to submit reports, but I'm not sure how to do that.